W-17876392 sap s/4hana odata limitation oauth2 (#2861)
mulesoft/docs-connectors: sap-s4hana-cloud/2.7/modules/ROOT/pages/sap-s4hana-cloud-connector-studio.adoc

  The user name used to initialize the session
  ** *Password* +
  The password used to authenticate the user
+
  * *OAuth 2 Application Connection Provider* +

>   In OAuth 2.0 authentication, a TLS context that comes with a truststore can't be added because the SDK does not expose the TLS context afterward. For OAuth 2.0 authentication, enter the following required information:

Reviewer: ```suggestion
  In OAuth 2.0 authentication, a TLS context that comes with a truststore can't be added because the SDK doesn't expose the TLS context afterward. For OAuth 2.0 authentication, enter this required information:
```
